### DISCUSSION TITLE
Plots lost

### DISCUSSION THREAD

**Discussion by Adam4600** [Chia-Network/chia-blockchain#6521] [Support]
After updating to 1.1.7 it stopped farming and all plots are invisible. I added plots location and still can not recognized not even single plot. 
![1](https://user-images.githubusercontent.com/85444879/120926921-e8f55980-c6ac-11eb-857e-75e27b99b521.png)

**Comment by gromran** [comment]
is the plot dir correctly added?

**Comment by Adam4600** [comment]
Yes it. It was working before an update.   
  

  

  

  
  
  
  
  
>

**Reply by MiT-NZ** [reply]
I have the same issue with you, roll back to 1.1.6, aslo I did test for a new plotting K32 still "Not found Plots", I think that is the bug which we only stay in 1.1.6 at moment.

**Comment by BrandtH22** [comment]
@Adam4600 , if you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
